x/zkism/types/metadata_test.go

```diff
@@ -224,6 +229,8 @@ func TestPublicInputsUnmarshalTrailingData(t *testing.T) {
 		TrustedStateRoot:   [32]byte{0x02},
 		NewHeight:          2,
 		NewStateRoot:       [32]byte{0x03},
+		Namespace:          [29]byte{0x04},
```

Comment:
This PR requires https://github.com/celestiaorg/celestia-zkevm-hl-testnet/pull/169 in order to rely no a raw 29 byte namespace here. 

For some reason the `Serialize` impl in celestia-types (rs) uses base64 encoding: https://github.com/eigerco/lumina/blob/main/types/src/nmt.rs#L576-L584